setup.py

```diff
@@ -66,7 +74,7 @@
         "hexbytes",  # Use same version as eth-ape
         "pydantic",  # Use same version as eth-ape
         # ** ApeWorX maintained **
-        "eth-ape>=0.6.1,<0.7",
+        "eth-ape>=0.5.0,<0.7",
```

Comment:
No, we shouldn't need to drop back to 0.5, perhaps you need to fetch core tags and re-install

```suggestion
        "eth-ape>=0.6.2,<0.7",
```